packages/flame/lib/src/collisions/hitboxes/polygon_hitbox.dart

```diff
@@ -8,6 +8,7 @@ class PolygonHitbox extends PolygonComponent
     with ShapeHitbox, PolygonRayIntersection {
   PolygonHitbox(
     super.vertices, {
+    super.position,
```

Comment:
Have you tried this? I don't think it will play along nicely with the vertices since the position is already set from the vertices.

Reply:
Yes I have, and it works as expected. Leaving position unmodified results in the hitbox's vertices being offset by topLeft compared to any parent. Providing a position of zero aligns it to the parent (when using the same vertices as the parent), any offset shows a visual offset by that amount - as position already does with CircleHitbox.

Reply:
Could you add a small test for this? :)

Reply:
Added (7f20899)